### ISSUE TITLE
Remove currently useless controls form the UI

### ISSUE THREAD

**Issue by vvasuki** [vedavaapi/ullekhanam-ui-v1#2]
The right side of the current page view contains some controls which have no use in the near future:

![image](https://user-images.githubusercontent.com/2664797/33465809-ca233812-d5fe-11e7-8e0c-9688a2743244.png)

They should be removed as non-functional controls make for bad UI. They can always be readded when there is better reason. @indictools - do you agree?

**Comment by indictools** [comment]
Let us do the UI cleanup once the functionality gets to some level of completion. We are not yet there.

**Comment by vvasuki** [comment]
I agree with what you said, but - "What to do?" is a separate question from "when to do it?".
